osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/CosemObjectAccessor.java

```diff
@@ -22,16 +22,18 @@
 import org.opensmartgridplatform.adapter.protocol.dlms.domain.factories.DlmsConnectionManager;
 import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ConnectionException;
 import org.opensmartgridplatform.adapter.protocol.dlms.exceptions.ProtocolAdapterException;
+import org.opensmartgridplatform.dlms.interfaceclass.attribute.AttributeClass;
+import org.opensmartgridplatform.dlms.interfaceclass.method.MethodClass;
 
 public class CosemObjectAccessor {
 
-    private static final String EXCEPTION_MSG_ACCESS_RESULT_NOT_SUCCESS =
-            "Access result not success but '%s'  while " + "writing attribute %s, classId %s, obisCode %s.";
-    private static final String EXCEPTION_MSG_WRITING_ATTRIBUTE =
-            "An exception occurred while writing attribute %s, " + "classId %s, obisCode %s.";
+    private static final String EXCEPTION_MSG_ACCESS_RESULT_NOT_SUCCESS = "Access result not success but '%s'  while "
```

Comment:
"Access result not success" doesn't read like proper English, consider rewrite to "Access result was not successful while writing attribute... " or something similar,

Reply:
Rephrased